README.md

```diff
@@ -174,10 +108,9 @@ with:
 * OpenTelemetry API version <!--OTEL_VERSION-->1.4.1<!--OTEL_VERSION-->
 * OpenTelemetry Instrumentation for Java version <!--OTEL_INSTRUMENTATION_VERSION-->1.4.0<!--OTEL_INSTRUMENTATION_VERSION-->
 
-## Correlating traces with logs
```

Comment:
Why this was removed? Also I am somewhat concerned about having documentation in `docs` folder which is not linked from anywhere.

Reply:
This is a specific feature/use case that is a bit outside of the scope of the README—we either put them all or none.

But we can reinstate it, with a link to the official docs if needed.

I understand the concern about the docs, but O11y docs should be the primary source that we link. We can use local docs for knowledge transfer while we think about automation.

Reply:
Can we leave a link to official docs in the README? Now that I think of it, there's a requirement in the GDI spec about that one specific use case:

```md
* MUST document how to configure log correlation
```

Reply:
I'll add it in my next commit.

Reply:
Fixed!